### DISCUSSION TITLE
LTS, releases cycle

### DISCUSSION THREAD

**Discussion by adrianbienias** [vercel/next.js#41906] [Help]
First of all, I want to say that Next.js is just great. Working with it makes web development a really great experience.
But, there's one thing that bothers me.
Next.js evolves and changes so rapidly that it's hard to bet on it in the long term.

I built a few apps with Next.js and currently, I have to maintain 3-4 different versions of Next.js.
Of course, I could upgrade those older apps to the latest version but if everything works I see no reason to just invest the time to update them just for the sake of chasing the newest version.

What if I would find a bug or a security flaw in versions 9, 10, or 11?
Do you still provide support for those versions or not?

Also, I'm an online tutor and I really enjoy teaching people how they can build apps using Next.js.
But If I invest my time to create a comprehensive video course about creating web apps with e.g. Next.js 12 how long it will be up to date?
I have students that still learn from my courses about AngularJS - the first version, recorded more than 10 years ago.
AngularJS LTS support ended at the end of 2021. It lasted 11 years.

Do you have any official release cycle and any plans for the LTS version?
Can I bet on some of your version and be sure you'll provide support for it for the next 5-10 years?

**Comment by adrianbienias** [comment]
I'm taking no answer as a negative - you don't plan any clear release cycle, nor LTS version.

To summary:

- In case of finding some security flaws in older versions, developers have to fix them on their own.

- When teaching others Next.js it's better not to make comprehensive courses because the next major version can drastically change the approach and make courses outdated.

**Comment by saveveltri** [comment]
is there any update on that ? it would be useful to have a per release end of support date. Thanks
